common/server/statshouse/statshouse-client.h

```diff
@@ -0,0 +1,38 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "add_metrics_batch.h"
+#include "common/smart_ptrs/singleton.h"
+
+namespace statshouse {
+
+class Client : vk::not_copyable {
```

Comment:
Naming `StatshouseClient` instead of `Client` is way better for text search / fuzzy search / method navigation

Reply:
fixed